src/main/java/com/simple/metrics/kafka/DropwizardReporter.java

```diff
@@ -0,0 +1,86 @@
+package com.simple.metrics.kafka;
```

Comment:
Is this a reasonable package name? Is `kafka-dropwizard-reporter` a reasonable name for the module?

https://github.com/apakulov/kafka-graphite defines its reporter class in `org.apache.kafka.common.metrics` (the same package as the `MetricsReporter` interface that this implements) in order to get access to some package-private methods. Is that considered bad form to patch new classes into an existing package you don't own, or is that okay? Would it be more natural to users of this package to access the class as `org.apache.kafka.common.metrics.DropwizardReporter`?